examples/basic_kv.go

```diff
@@ -0,0 +1,212 @@
+package main
+
```

Comment:
Could you put a comment with a link to the page for these examples?

```Go
// http://docs.basho.com/riak/latest/dev/using/basics/
```

Thanks!!